addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -143,18 +142,10 @@ export class Chatbot extends Record {
      * @returns {Promise<boolean>} Whether the script is ready to go to the next step.
      */
     async _processAnswerQuestionSelection(message) {
-        if (this.currentStep.selectedAnswer) {
+        const answer = this.currentStep.selectedAnswer;
+        if (answer) {
             return true;
         }
-        const answer = this.currentStep.answers.find(({ name }) =>
-            message.body.includes(escape(name))
-        );
-        this.currentStep.selectedAnswer = answer;
-        await rpc("/chatbot/answer/save", {
```

Comment:
this route can be removed?

Reply:
Not in stable :) But yes, I'll get rid of it in master!